source/extensions/filters/common/rbac/utility.cc

```diff
@@ -11,6 +15,15 @@ RoleBasedAccessControlFilterStats generateStats(const std::string& prefix, Stats
   return {ALL_RBAC_FILTER_STATS(POOL_COUNTER_PREFIX(scope, final_prefix))};
 }
 
+std::string responseDetail(const std::string& policy_id) {
```

Comment:
Actually I think this is useful enough we may want to put it as a StreamInfo utility, as I'm not sure they're all whitespace safe.
You can apply it manually to rbac details today and add a TODO(alyssawilk) to apply it to all details, SG?

Reply:
thanks that makes sense, updated the TODO.